back-end/controllers/main-controller.js

```diff
@@ -0,0 +1,50 @@
+const connection = require('../db');
+
+const getAllOrders = (req, res) => {
+	//soft delete active
+	//const query = `SELECT * FROM orders where is_deleted = ?`;
+	//const data = [orders.is_deleted];
+	const query = `SELECT * FROM orders`;
+	connection.query(query, (err, result) => {
+		if (err) throw err;
+		res.json(result);
+	});
+};
+
+const addNewOrder = (req, res) => {
+	const query = `INSERT INTO orders (ordersId)
+	 VALUES (?) `;
```

Comment:
Why are you adding only one value? and why it is the id?

Reply:
there is nothing in orders table except the id